app/api/router.py

```diff
@@ -2,9 +2,18 @@
 
 from api.example.views import router as example_router
 from api.system.views import router as system_router
-from api.search.views import router as search_router
+ master
```

Comment:
Are these 'master' lines artifacts from a merge or did you mean to add them? Please remove them if you did not mean to. If you did mean to, why are they necessary?